internal/future/statestore/public.go

```diff
@@ -59,10 +59,10 @@ func New(cfg config.View) (Service, error) {
 	return newRedis(cfg)
 }
 
-// GetDeprecatedRedisPool returns a Redigo Redis pool.
+// getDeprecatedRedisPool returns a Redigo Redis pool.
```

Comment:
Why are we making this lower case? I thought this was meant to be publicly exposed to legacy clients?

Reply:
Are you referring to examples/backendclient? That one does not talk to redis directly. The latest change that you made last week abstract the redis existence so we only need to expose this function package-wise

Reply:
Is this used in code at all anymore? This was used - and hence it was in capital.. but i checked and does not seem so.. remove this method completely if not used,